### DISCUSSION TITLE
Can This be done?

### DISCUSSION THREAD

**Discussion by CaptRR** [AzuraCast/AzuraCast#3875] [Q&A]
Sorry if this is the wrong place.  If so please remove.

So I was wondering if I could use Azuracast in a very certain situation, but I am not sure.

Here is the situation.  For most of the time I run my stream from my house using a commercial automation system.  It does a good job.  However getting DJ's and schduled shows setup and deleted can be a bit of a pain.  Currently I edit the liquidsoap script on my server to add and remove DJ's.  Scheduling the shows is done manually in the music scheduling system.

With that background here is what I was wondering.   Basically I was wondering if I could make Azura cast take over whenever a DJ connects or when a scheduled show plays, but when nothing else is on drop back to the automation at my house.  That would let me use my automation system (which does a great job for automated radio), but still make it easy to schedule and add/remove DJ's as they come and go.

Thanks

**Comment by BusterNeece** [comment]
AzuraCast is meant to have its own AutoDJ be the fallback for live streams and other broadcasts. It wouldn't likely be possible to have it only turn on when a live DJ is connected to its stream. This would generally be an advanced function that would be better suited by a Liquidsoap script that's running on a standalone basis.
